### ISSUE TITLE
env / default.properties logs_directory setting is ignored

### ISSUE THREAD

**Issue by kuechlerm** [getgauge/gauge#1135]
### Expected behavior
The logs_directory setting in env/default.properties should be used by gauge. 

### Actual behavior
The logs go into a folder called "logs" no matter what.

### Steps to reproduce
1. create a gauge project
2. change env/default.properties logs_directory
3. gauge run

### Gauge version
```
Gauge version: 1.0.0
Commit Hash: 5a99965

Plugins
-------
csharp (0.10.4)
dotnet (0.1.0)
html-report (4.0.4)
js (2.3.1)
screenshot (0.0.1)
```

**Comment by ajoecker** [comment]
it is getting even more problematic if a later process access the logs_directory environment variable to do things (like a plugin to log also there), this fails then with an exception, as the given directory is not existing

**Comment by BugDiver** [comment]
This issue has been fixed through #1136. Should be available in nightly >= 2-08-2018
